content/en/docs/concepts/workloads/controllers/replicationcontroller.md

```diff
@@ -202,19 +202,6 @@ As mentioned above, whether you have 1 pod you want to keep running, or 1000, a
 
 The ReplicationController makes it easy to scale the number of replicas up or down, either manually or by an auto-scaling control agent, by simply updating the `replicas` field.
 
-### Rolling updates
-
-The ReplicationController is designed to facilitate rolling updates to a service by replacing pods one-by-one.
-
-As explained in [#1353](http://issue.k8s.io/1353), the recommended approach is to create a new ReplicationController with 1 replica, scale the new (+1) and old (-1) controllers one by one, and then delete the old controller after it reaches 0 replicas. This predictably updates the set of pods regardless of unexpected failures.
-
-Ideally, the rolling update controller would take application readiness into account, and would ensure that a sufficient number of pods were productively serving at any given time.
-
-The two ReplicationControllers would need to create pods with at least one differentiating label, such as the image tag of the primary container of the pod, since it is typically image updates that motivate rolling updates.
-
-Rolling update is implemented in the client tool
-[`kubectl rolling-update`](/docs/reference/generated/kubectl/kubectl-commands#rolling-update). Visit [`kubectl rolling-update` task](/docs/tasks/run-application/rolling-update-replication-controller/) for more concrete examples.
-
```

Comment:
Is `kubectl rolling-update` under the Deployment section on this page still relevant?

Reply:
@surajssd, to fully clean-up, can you re-check the sections under the following Sub-headers on this page:

`Deployment`, remove `kubectl rolling-update`?
`Deleting just a ReplicationController`, there is a link that no longer works because you removed the content?